docs/js/features/connectivity/bas-external-system.md

```diff
@@ -0,0 +1,165 @@
+      "preLaunchTask": "npm: build",
+      "outFiles": [
+        "${workspaceFolder}/dist/**/*.js"
+      ]
+    }
+  ]
+}
+```
+In this document we use a simple Nest application but for any express or [CAP project](https://community.sap.com/topics/cloud-application-programming) it will look similar.
+The entries are quite self-explanatory, the `program` and `preLaunchTask` will be of course different depending on your application.
+The relevant thing will happen in the `.env` file which is read when starting the application.
+If you do not have a `.env` file create one or adjust the existing one.
+Just add the following entry to the `.env` file:
+
+```
+destinations="[{"name": "<destinationName>","url": "<destinationUrl>"}]"
+```
+Fill in the `name` and `url` you have noted down from the CF destination you want to connect to.
+Once the request reaches the CF account via the proxy, it reads all authorization information from the real destination.
+All requests done with the SDK will now reach the S/4 system. 
+You can start your application via the launch button. 
+
+:::note
+The SDK also offers a `mockTestDestination()` method which reads destination information from a `system.json` and `credentials.json` file.
+In the end, this only sets the `destinations` environment variables as the `.env` file does, but could become advantageous if you have many systems.
+You can also set the `credentials.json` to your git ignore list so that they are not checked in by accident and share the systems with your colleagues.
+:::
+
+For the simple case you would add `mockTestDestination(<destinationName>)` to your local startup script and have a `systems.json` in your project root:
+```JSON
+{
+    "systems": [
+        {
+            "alias": "your CF destination name",
+            "uri": "your CF destination url "
+        }
+    ]
+}
+```
+
+## Closing Remarks for Cloud Systems
+
+In case you have a cloud system like S/4 Hana cloud or any other system on the internet you can follow the same approach.
+Just add values for `user` and `password` to the environment variables if the system requires authorization.
+In case you use the `mockTestDestination()` add the login information to the `credentials.json`.
+
+:::note
+Via the connection between the BAS and CF it is also possible to import all environmentvariabless from the CF account to the BAS.
+In particular the `VCAP_SERVICE` variable containing all service information.
+If you import these you can do the real destination via the destination service in your locally deployed app.
+This works for all destination with a proxy type `internet` but not for the ones with `onPremise`. 
+The reason for this is the interference of two proxies: (1) The web proxy of the BAS and (2) the connectivity proxy in CF.
+If you load an onPremise destination via the destination service it will contain the connectivity proxy of CF, although you would need the web proxy when you run locally in BAS.
```

Comment:
```suggestion
If you load an On-Premise destination via the destination service it will contain the connectivity proxy of CF, although you would need the web proxy when you run locally in BAS.
```